scripts/test-react-native-setup.sh

```diff
@@ -13,7 +15,11 @@ buck build ReactAndroid/src/main/java/com/facebook/react/shell
 
 echo "Build React Native via Gradle"
 yarn install
-./gradlew --no-daemon :ReactAndroid:packageReactNdkLibsForBuck
 
-echo "Assemble RNTester app"
-./gradlew --no-daemon :packages:rn-tester:android:app:assembleRelease
+./gradlew :ReactAndroid:assembleDebug
+
+# ./gradlew --no-daemon :ReactAndroid:packageReactNdkLibsForBuck
+
+# echo "Assemble RNTester app"
+# ./gradlew --no-daemon downloadAll
+# ./gradlew --no-daemon :packages:rn-tester:android:app:assembleRelease
```

Comment:
Please note that those are still needed

Reply:
Yeap, but run all those will timeout on github action.
You can see https://github.com/react-native-community/docker-android/actions/runs/2745390327. 6 hours still not finished.

Reply:
Mmm that's really odd. I can't see on which Gradle task it got stuck actually

Reply:
For Android not. Also Hermes needs to be built at a certain point and the Docker image should contain all the necessary tooling to build it so I won't try to use prebuilts here

Reply:
> For Android not. Also Hermes needs to be built at a certain point and the Docker image should contain all the necessary tooling to build it so I won't try to use prebuilts here

If we build from source, the github action will just timeout. Not sure there is a better way (Ninja on android or limit cpu arch on build arm64 maybe)?
cc @Huxpro

Reply:
> Not sure there is a better way 

The easiest would be to build per arch and split the job in 4